source/_components/epsonprinter.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: "Epson Printer"
+description: "Instructions on how to integrate Epson Workforce Printer into Home Assistant."
+date: 2019-04-09 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Sensor
+ha_release: 0.92
+ha_iot_class: Local Polling
+---
+
+The `epson printer` platform allows you to monitor the ink levels of a Epson Workforce printer from Home
+Assistant.
+
+To add Epson Printer to your installation,
+add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
```

Comment:
Why is it using media_player?

Reply:
It isn't, it is is using sensor, changed it